extra-functions/extra-03/script.js

```diff
@@ -0,0 +1,8 @@
+function isLongerThan(number, text) {
+  if (text.length > number) {
```

Comment:
Just as a sidenote, this could also be implemented by doing `return text.length > number`

I prefer the way you did it though as it is more readable